controllers/cloud.redhat.com/providers/iqe/impl.go

```diff
@@ -244,7 +244,8 @@ func CreateIqeJobResource(cache *rc.ObjectCache, cji *crd.ClowdJobInvocation, en
 		"ignore-check.kube-linter.io/no-liveness-probe":  "probes not required on Job pods",
 		"ignore-check.kube-linter.io/no-readiness-probe": "probes not required on Job pods",
 	}
-	utils.UpdatePodTemplateAnnotations(&j.Spec.Template, annotations)
+	utils.UpdateAnnotations(&j.Spec.Template, annotations)
```

Comment:
I still think this is odd, can you explain does it look for annotations on the Job directly?

Reply:
Yes, it looks on the job definition even if it is not getting invoked, a 'deployment' of said job gets created - and since we don't define probes for it, it will alert us